### ISSUE TITLE
Reproducibility Of Results

### ISSUE THREAD

**Issue by rhys-newbury** [lis-epfl/apg_trajectory_tracking#7]
I am trying to run the 'train_drone.py' script, and the training does not seem to converge in any meaningful way, with the stables runs staying at 0%. I just run `python train_drone.py` with the default parameters. I was able to use the trained model and saw good results. Is there any tricks to help with training. 

I am using these versions of libraries:

        'torch==1.13.1', 
        'gym==0.21.0',
        'numpy==1.23.4', 
        'matplotlib==3.8.3', 
        'scipy==1.12.0', 
        'pyglet==1.5.27', 
        'ruamel.yaml==0.18.6', 
        'tqdm==4.66.2', 
        'casadi==3.6.4', 
        'pandas==2.2.1',
        'scikit-learn==1.4.1.post1',
        'pyquaternion==0.9.9',
        'tensorboard==2.16.2'

**Comment by rhys-newbury** [comment]
Results on default settings:

Epoch 398
Loss (controller): 23.35
Data used for training: 796000
rand: Average div of full runs: nan (nan)
Ratio of stable runs: 0.00
Save model with score  251.0
[232.9, 231.9, 229.7, 232.2, 239.6, 239.9, 240.1, 238.9, 238.5, 240.9, 241.4, 244.4, 243.4, 242.0, 242.8, 247.1, 245.1, 245.9, 245.5, 246.6, 244.3, 245.3, 246.3, 247.0, 248.8, 246.0, 248.2, 247.7, 248.7, 249.2, 249.3, 249.8, 249.2, 248.1, 246.1, 250.3, 248.7, 249.6, 250.0, 250.3, 250.5, 250.2, 250.5, 249.8, 249.7, 250.8, 250.3, 249.9, 249.6, 250.6, 250.2, 250.4, 250.4, 250.6, 250.7, 250.3, 250.6, 250.5, 250.7, 250.6, 250.7, 250.7, 250.7, 250.7, 250.8, 250.8, 250.4, 250.7, 250.7, 250.7, 250.7, 250.7, 250.7, 250.8, 250.7, 250.9, 250.7, 250.7, 250.9, 250.8, 250.9, 250.9, 250.7, 250.9, 250.9, 250.9, 250.7, 250.9, 250.9, 250.8, 250.9, 251.0, 250.9, 250.8, 250.6, 250.9, 250.9, 250.9, 250.8, 250.9, 250.9, 250.9, 250.8, 250.8, 250.8, 250.7, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.8, 251.0, 250.9, 250.9, 250.9, 250.9, 250.9, 251.0, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 251.0, 250.9, 250.9, 251.0, 250.9, 251.0, 250.6, 250.9, 251.0, 250.9, 250.9, 251.0, 250.9, 251.0, 251.0, 250.9, 251.0, 250.9, 251.0, 250.9, 250.9, 250.9, 250.9, 251.0, 250.9, 250.9, 251.0, 250.9, 250.9, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 250.9, 250.9, 250.9, 250.7, 250.9, 250.9, 250.8, 250.9, 251.0, 250.9, 250.8, 250.6, 250.9, 250.9, 250.9, 250.8, 250.9, 250.9, 250.9, 250.8, 250.8, 250.8, 250.7, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.8, 251.0, 250.9, 250.9, 250.9, 250.9, 250.9, 251.0, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 250.9, 251.0, 250.9, 250.9, 251.0, 250.9, 251.0, 250.6, 250.9, 251.0, 250.9, 250.9, 251.0, 250.9, 251.0, 251.0, 250.9, 251.0, 250.9, 251.0, 250.9, 250.9, 250.9, 250.9, 251.0, 250.9, 250.9, 251.0, 250.9, 250.9, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 250.9, 251.0, 250.9, 251.0, 250.9, 251.0, 250.9, 250.9, 251.0, 250.9, 251.0, 250.9, 251.0, 250.9, 251.0, 251.0, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 251.0, 251.0, 250.9, 251.0, 250.9, 251.0, 250.9, 250.9, 251.0, 251.0, 251.0, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 250.9, 251.0, 250.9, 251.0, 251.0, 251.0, 251.0, 250.9, 251.0, 250.9, 251.0, 251.0, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 250.9, 251.0, 250.9, 251.0, 251.0, 251.0, 250.9, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0, 251.0,
